### PR TITLE
O16N lgbm criteo

### PR DESCRIPTION
**Pull request by miguelgfierro**
### Description

Same code as https://github.com/Microsoft/Recommenders/pull/680/commits/c4ba0dca62c1488deaea8eb3dd88dade8556ee7e
Splitting this PR https://github.com/Microsoft/Recommenders/pull/680 into 2

### Checklist:

- [ ] My code follows the code style of this project, as detailed in our [contribution guidelines](../CONTRIBUTING.md).
- [ ] I have added tests.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "\n",
+    "Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Goal\n",
```
Comment (yueguoguo) [inline notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: Maybe a better title is needed here.

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+   ],
+   "source": [
+    "ws = Workspace.from_config('/dbfs/aml_config/config.json')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Serialized Model\n",
```
Comment (yueguoguo) [inline notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: Section number starts from 2. 1 is missing.

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb [hunk 3]
```diff
@@ -0,0 +1,785 @@
...
+       "</div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "model_local = os.path.join(os.getcwd(), MODEL_NAME)\n",
+    "dbutils.fs.cp('dbfs:/' + MODEL_NAME, 'file:' + model_local, recurse=True)"
```
Comment (yueguoguo) [inline notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: Will this (using of `dbutils`) work out of box in the notebook?

Comment (gramhagen) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: only on databricks, but that's one of the assumptions listed at the top

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb (hunk 2 above)
Comment (gramhagen) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: good catch, just going to remove numbering

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb (hunk 1 above)
Comment (gramhagen) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: ok, updating this section a bit

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb (hunk 3 above)
Comment (yueguoguo) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: I see 👍

File: notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb (hunk 1 above)
Comment (miguelgfierro) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: we can add the svg that was going to the document. I think @jreynolds01 is going to work on this PR this week, I can update it tomorrow

Comment (miguelgfierro) [reply notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb]: The diagram is already updated

**Comment by miguelgfierro** [comment]
@jreynolds01 merging then, I will update the vanity url so the notebook is linked to staging

**Pull request merged**